searxng/searxng
# Theme: remove __common__ directories

## What does this PR do?

* Remove the `__common__` directories.
* Remove `plugins/js/*` and `plugins/css/*` files.
* Remove `searx.webapp.global_favicons` : they were used in the oscar theme.
* `/autocompleter` : the string results are always escaped.

## Why is this change important?

One step closer to move the theme source to the `src` directory.

## How to test this PR locally?

check the following item works as expected:
* the button "Submit a new issue on Github including the above information"
* the bar charts on `/stats`
* `/favicon.ico`
* `/opensearch.xml` with and without autocompletion
* autocompletion
* the "Vim-like hotkeys" preference
* the "Search on category select" preference

## Author's checklist



## Related issues
